app/api/v2/managers/contact_api_manager.py

```diff
@@ -0,0 +1,15 @@
+from app.api.v2.managers.base_api_manager import BaseApiManager
+from app.api.v2.responses import JsonHttpNotFound
+
+
+class ContactApiManager(BaseApiManager):
+    def __init__(self, data_svc, file_svc, contact_svc):
+        super().__init__(data_svc=data_svc, file_svc=file_svc)
+        self.contact_svc = contact_svc
+
+    def get_contact_report(self, contact: str = None):
+        if contact == 'http':
+            contact = contact.upper()
```

Comment:
would it be better to just keep the dictionary keys uppercase all the time? We could also enforce that in the `report` decorator function definition in contact_svc

Reply:
that would be much better! I was having trouble figuring out where the dictionary values were being stored, but this has been updated to standardize stored contact keys as uppercase. thanks!